src/components/sidebar/routesView/routeShow.scss

```diff
@@ -45,7 +45,11 @@
 
         .toggleTitle {
             display: inline-block;
-            width: 110px;
+            width: 270px;
```

Comment:
Static width is not a good solution (I know it was before a static width as well). But here is the point: if it wasn't a static width, you wouldn't have had to change it here again. So avoid using static widths :P (of course there are exceptions...)

Reply:
I guess why this previously was static is that the toggle button wanted to be floating at a specific point. This change sets the toggle to be in the right side, so I could indeed see if we could remove this static thing.

Reply:
this is fixed